submissions/github-bot/readme.md

```diff
@@ -0,0 +1,60 @@
+# Chimoney Github Bot
+
+
+The Environment variables (.env) file contains:
+
+- `PORT`: Port on which the bot's HTTP server will run.
```

Comment:
Since this is a Github workflow, could you document where these envs are added

And also outline step by step how any Github repo maintainer can add this workflow to their own repo. Make it super simple for the Maintainers to use

Reply:
> Since this is a Github workflow, could you document where these envs are added
> 
> And also outline step by step how any Github repo maintainer can add this workflow to their own repo. Make it super simple for the Maintainers to use

Before the bot can be used, it had to be published to github action marketplace.

Then it can be used like this:

```yaml
on: issue_comment 
  
 name: Chimoney 
  
 permissions: 
   issues: write 
   contents: read 
  
 jobs: 
     runs-on: ubuntu-latest 
     strategy: 
       matrix: 
         node-version: [18.x] 
  
     steps: 
       - name: Check for Command 
         id: command 
         uses: chimoney/github-chimoney-bot@v1
            env:
          PORT: ${{ secrets.PORT }}
          BOT_URL: ${{ secrets.BOT_URL }}
          API_KEY: ${{ secrets.API_KEY }}
          REDIRECT_URL: ${{ secrets.REDIRECT_URL }}
          CHIMONEY_BASE_URL: ${{ secrets.CHIMONEY_BASE_URL }}
          CHIMONEY_BASE_HOST: ${{ secrets.CHIMONEY_BASE_HOST }}
          DEFAULT_EMAIL: ${{ secrets.DEFAULT_EMAIL }}
           
```

In the GitHub repository, go to the "Settings" tab then select "Secrets."
Click on the "New repository secret" button for each of the environment variables used in the workflow. Name them exactly as used in the workflow YML file.

Reply:
![IMG_20231030_093611.jpg](https://github.com/Chimoney/chimoney-community-projects/assets/62471060/3c780115-d4f8-4085-9659-aadc9815371a)

Reply:
Please update your readme to include these instructions because you don’t want every maintainer to first ask or email you before setting this action up

Reply:
Done